static/js/stats.js

```diff
@@ -47,7 +47,7 @@ function countMarkers(map) { // eslint-disable-line no-unused-vars
                 pokeCounts.push(
                     [
                         '<img src=\'static/icons/' + pkmnCount[i].ID + '.png\' />',
-                        '<a href=\'http://www.pokemon.com/us/pokedex/' + pkmnCount[i].ID + '\' target=\'_blank\' title=\'View in Pokédex\' style=\'color: black;\'>' + pkmnCount[i].Name + '</a>',
+                        '<a href=\'https://pokemon.gameinfo.io/en/pokemon/' + pkmnCount[i].ID + '\' target=\'_blank\' title=\'View in Pokédex\' style=\'color: black;\'>' + pkmnCount[i].Name + '</a>',
```

Comment:
when the site already support locale, **/en/** why not using the locale from config?
This way the user would get straight to the right locale site for him.
(And the sites support many locales)